documentation-site/components/yard/config/header-navigation.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  HeaderNavigation,
+  ALIGN,
+  StyledNavigationItem,
+  StyledNavigationList,
+} from 'baseui/header-navigation';
+import {StyledLink} from 'baseui/link';
+import {Button} from 'baseui/button';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const HeaderNavigationConfig: TConfig = {
+  imports: {
+    'baseui/header-navigation': {
+      named: ['HeaderNavigation', 'ALIGN'],
+    },
+    'baseui/link': {
+      named: ['StyledLink'],
+    },
+    'baseui/button': {
+      named: ['Button'],
+    },
+  },
+  scope: {
+    HeaderNavigation,
+    ALIGN,
+    StyledNavigationItem,
+    StyledNavigationList,
+    StyledLink,
+    Button,
+  },
+  theme: ['headerNavigationFill'],
+  props: {
+    children: {
+      value: `<StyledNavigationList $align={ALIGN.left}>
+<StyledNavigationItem>Uber</StyledNavigationItem>
+</StyledNavigationList>
+<StyledNavigationList $align={ALIGN.center} />
+<StyledNavigationList $align={ALIGN.right}>
+<StyledNavigationItem>
+<StyledLink href="#basic-link1">Tab Link One</StyledLink>
+</StyledNavigationItem>
+<StyledNavigationItem>
+<StyledLink href="#basic-link2">Tab Link Two</StyledLink>
+</StyledNavigationItem>
+</StyledNavigationList>
+<StyledNavigationList $align={ALIGN.right}>
+<StyledNavigationItem>
+<Button>Get started</Button>
+</StyledNavigationItem>
+</StyledNavigationList>
```

Comment:
(welp, we'll probably need to do something about the initial knob formatting through prettier)

Reply:
maybe we could mark a prop to take up the full width of the container? (span both columns)

Reply:
Yea, it's just there's not much value to edit the children knob vs edit the code snippet at that point.

Reply:
good point